dali/operators/reader/loader/loader.cc

```diff
@@ -19,43 +19,59 @@ namespace dali {
-this parameter is ignored.)code", 1024)
+      R"code(Size of the buffer that is used for shuffling.
+
+If ``random_shuffle`` is turned off,  this parameter is ignored.)code", 1024)
   .AddOptionalArg("num_shards",
-      R"code(Partition the data into this many parts (used for multiGPU training).)code", 1)
+      R"code(Partitions the data into the specified number of parts (shards). This is typically
+used for multi-GPU or multi-node training.)code", 1)
   .AddOptionalArg("shard_id",
-      R"code(Id of the part to read.)code", 0)
+      R"code(Index of the shard to read.)code", 0)
   .AddOptionalArg("tensor_init_bytes",
       R"code(Hint for how much memory to allocate per image.)code", 1048576)
   .AddOptionalArg("stick_to_shard",
-      R"code(Whether reader should stick to given data shard instead of going through the whole dataset.
-When decoder caching is used, it reduces significantly the amount of data to be cached, but could affect
-accuracy in some cases)code", false)
+      R"code(Determines whether the reader should stick to a data shard instead of going through
+the entire dataset.
+
+If you use the decoder caching, it significantly reduces the amount of data to be cached, but
+might affect accuracy.)code", false)
   .AddOptionalArg("read_ahead",
-      R"code(Whether accessed data should be read ahead. In case of big files like LMDB,
-RecordIO or TFRecord it will slow down first access but will decrease the time of all following
-accesses.)code", false)
+      R"code(Determines whether the accessed data should be read ahead.
+
+For large files such as LMDB, RecordIO, or TFRecord, this argument slows down the first access but
+decreases the time of all of the following accesses.)code", false)
   .AddOptionalArg("prefetch_queue_depth",
-      R"code(Specifies the number of batches prefetched by the internal Loader. To be increased when pipeline
-processing is CPU stage-bound, trading memory consumption for better interleaving with the Loader thread.)code", 1)
+      R"code(Specifies the number of batches to be prefetched by the internal Loader.
+
+This value should be increased when the pipeline is CPU-stage bound, trading memory
+consumption for better interleaving with the Loader thread.)code", 1)
   .AddOptionalArg("skip_cached_images",
-      R"code(If set to true, loading data will be skipped when the sample is present in the decoder cache.
-In such case the output of the loader will be empty)code", false)
+      R"code(If set to True, the loading data will be skipped when the sample is
+in the decoder cache.
+
+In this case, the output of the loader will be empty.)code", false)
   .AddOptionalArg("lazy_init",
-      R"code(If set to true, Loader will parse and prepare the dataset metadata only during the first `Run`
-instead of in the constructor.)code", false)
+      R"code(If set to True, the Loader parses and prepares the dataset metadata only during the
```

Comment:
I'm not sure if this is clear enough.
```suggestion
      R"code(Parse and prepare the dataset metadata only during the first run instead of in the constructor.
```

Reply:
Done